packages/seacas/libraries/ioss/src/Ioss_GroupingEntity.C

```diff
@@ -327,3 +327,165 @@ void Ioss::GroupingEntity::property_update(const std::string &property,
     nge->property_add(Ioss::Property(property, value));
   }
 }
+
+bool Ioss::GroupingEntity::equal(const Ioss::GroupingEntity &rhs)
```

Comment:
Why?

Reply:
Moved to branch `tkordenbrock-seacas/add-ioshell-compare`.